src/styles/_search-bar.scss

```diff
@@ -0,0 +1,42 @@
+#search-bar-div {
+  position: absolute;
+  left: 33%;
+  top: 15px;
+  width: 35%;
+  z-index: 600;
+}
+
+@media (max-width: 640px) {
+  #search-bar-div {
+    left: 2%;
+    width: 70%;
+  }
+}
+
+#search-bar {
```

Comment:
@jimboweb Also perhaps add `background-clip: padding-box;` to keep is more consistent.

Reply:
added it